update-chart-versions.sh

```diff
@@ -0,0 +1,168 @@
+#!/bin/bash
+
+function displayHelp {
+  printf "usage: ./update-chart-versions.sh [OPTIONS]
+          -h, --help
+            Display help
+          [OPTIONAL]
+          -m, --master
```

Comment:
I suppose we can, I went with the approach we took before where the default is dev. 
But yeah, I think it would make more sense. We can then default to dev in our launch-or-update-local.sh